python/plugins/db_manager/db_plugins/data_model.py

```diff
@@ -177,7 +177,7 @@ def modelDone(self):
 class SqlResultModelTask(QgsTask):
 
     def __init__(self, db, sql, parent):
-        super().__init__()
+        super().__init__(description=QApplication.translate("DBManagerPlugin","DB Manager executing SQL"))
```

Comment:
Can we drop the "dbmanager.." and just say "executing SQL"? I think don't think it's needed here